test/manual/amp-lightbox-carousel-demo.amp.html

```diff
@@ -6,6 +6,7 @@
   <link rel="canonical" href="amps.html" >
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-image-viewer" src="https://cdn.ampproject.org/v0/amp-image-viewer-0.1.js"></script>
```

Comment:
this should not need to load the image viewer. lightbox should load the extension.